crs_utils_py/setup.py

```diff
@@ -0,0 +1,44 @@
+import os
+from glob import glob
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'crs_utils_py'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    # Packages to export
+    packages=[],
```

Comment:
In order to get the executable to install properly I had to change line 12 to `packages=find_packages(),`